hack/toolchain_install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+# parse input
+BRANCH=${BRANCH:-pubviewer-mvp}
+export QUAY_NAMESPACE=${QUAY_NAMESPACE:-workspaces}
+TAG=${1:-e2etest}
+
+# create a temporary direction
+f=$(mktemp --directory /tmp/toolchain.XXXX)
+cd "$f"
+
+# checkout
+git clone --depth 2 https://github.com/codeready-toolchain/member-operator.git
+git clone --depth 2 --branch "${BRANCH}" https://github.com/filariow/toolchain-e2e.git
+git clone --depth 2 --branch "${BRANCH}" https://github.com/filariow/host-operator.git
+git clone --depth 2 --branch "${BRANCH}" https://github.com/filariow/toolchain-common.git
+git clone --depth 2 --branch "${BRANCH}" https://github.com/filariow/toolchain-api.git api
+git clone --depth 2 --branch "${BRANCH}" https://github.com/filariow/registration-service
+
+# deploy
+( 
+  kubectl create namespace toolchain-host-operator --dry-run=client --output=yaml | \
+    kubectl apply -f -
+  kubectl create namespace toolchain-member-operator --dry-run=client --output=yaml | \
+    kubectl apply -f -
```

Comment:
We have a mix of `kubectl` and `oc` invocations, even though I think everything is covered by `kubectl` directly.  I'd prefer we use one tool and stick to it.